reviews/review-priorities.md

```diff
@@ -0,0 +1,68 @@
+# PR Prioritisation
+
+The Mbed TLS core team prioritise PRs to help us spend our review capacity on the items that best help the project.
+
+When PRs are first triaged, a label is applied to help reviewers determine what to work on (and what not to work on). The different classifications are described below.
```

Comment:
What's the advantage of labels, compared to columns on a board?

Reply:
As discussed earlier: there are a few ways to capture this kind of thing in GitHub.

Labels are an obvious choice because they are simple, easily visible to people viewing the PR, quick to set, etc. They also interact well with our existing boards - no other changes are required to our setup. They are easy for people who don't know the process well to observe and make sense of. It's easy to filter to find the high-priority items in any GitHub context (e.g., searching PRs, viewing a board).

You suggested capturing this via a column on the unified board, which I don't think works so well. The unified board tracks PR/issue _progress_, not priority. We'd need to add extra columns for the various priorities, for both review and (less critically) stalled columns - if done thoroughly it would add a lot of complexity to this board. Outside / casual contributors may not know to look at the board so might not follow the prioritisation. It also wouldn't be visible in the epics board, PR search results, etc.

The main argument against using labels for this seems to be that we have too many labels (57 now). This is very true and I support simplifying the set of labels. E.g., there are quite a few relating to the area affected (component: <X>, TLS 1.3/MPS, PSA, etc), which we could drop; also some obsolete ones relating to things like CLA, baremetal, etc. There are some that are hardly used (e.g., Information, notabug, regression, wontfix) which could go and just be replaced by an appropriate comment in the text of the PR/issue.

Reply:
> This is very true and I support simplifying the set of labels.

The thing is, we all agree on that, but we can't agree about which labels to remove. Different labels are useful to different people.

Of the ones you're listed, the only ones I agree with are the obsolete ones, where we could make a list of labelled PR/issues once and for all and then remove the labels.